packages/integration/lib/src/widgets/user/user_created_comment_listview.dart

```diff
@@ -0,0 +1,787 @@
+import 'package:infinite_scroll_pagination/infinite_scroll_pagination.dart';
+import 'package:likeminds_feed_flutter_core/likeminds_feed_core.dart';
+import 'package:likeminds_feed_flutter_core/src/bloc/user_created_comment/user_created_comment_bloc.dart';
+import 'package:likeminds_feed_flutter_core/src/utils/constants/assets_constants.dart';
+import 'package:likeminds_feed_flutter_core/src/utils/constants/post_action_id.dart';
+import 'package:likeminds_feed_flutter_core/src/utils/persistence/user_local_preference.dart';
+import 'package:likeminds_feed_flutter_core/src/utils/typedefs.dart';
+import 'package:likeminds_feed_flutter_core/src/views/media/media_preview_screen.dart';
+import 'package:likeminds_feed_flutter_core/src/views/post/widgets/delete_dialog.dart';
+import 'package:media_kit_video/media_kit_video.dart';
+import 'package:overlay_support/overlay_support.dart';
+
+class LMFeedUserCreatedCommentListView extends StatefulWidget {
+  const LMFeedUserCreatedCommentListView({
+    Key? key,
+    required this.uuid,
+    this.postBuilder,
+    this.emptyFeedViewBuilder,
+    this.firstPageLoaderBuilder,
+    this.paginationLoaderBuilder,
+    this.feedErrorViewBuilder,
+    this.noNewPageWidgetBuilder,
+  }) : super(key: key);
+
+  // The user id for which the user feed is to be shown
+  final String uuid;
+  // Builder for post item
+  // {@macro post_widget_builder}
+  final LMFeedPostWidgetBuilder? postBuilder;
+  // {@macro context_widget_builder}
+  // Builder for empty feed view
+  final LMFeedContextWidgetBuilder? emptyFeedViewBuilder;
+  // Builder for first page loader when no post are there
+  final LMFeedContextWidgetBuilder? firstPageLoaderBuilder;
+  // Builder for pagination loader when more post are there
+  final LMFeedContextWidgetBuilder? paginationLoaderBuilder;
+  // Builder for error view when error occurs
+  final LMFeedContextWidgetBuilder? feedErrorViewBuilder;
+  // Builder for widget when no more post are there
+  final LMFeedContextWidgetBuilder? noNewPageWidgetBuilder;
+
+  @override
+  State<LMFeedUserCreatedCommentListView> createState() =>
+      _LMFeedUserCreatedCommentListViewState();
+}
+
+class _LMFeedUserCreatedCommentListViewState
+    extends State<LMFeedUserCreatedCommentListView> {
+  ValueNotifier<bool> rebuildPostWidget = ValueNotifier(false);
+  int _page = 1;
+  static const int pageSize = 10;
+  final PagingController<int, LMCommentViewData> _pagingController =
+      PagingController(firstPageKey: 1);
+  bool userPostingRights = true;
+  LMFeedThemeData? feedThemeData;
+  final ValueNotifier postUploading = ValueNotifier(false);
+  final LMFeedUserCreatedCommentBloc _userCreatedCommentBloc =
+      LMFeedUserCreatedCommentBloc();
+  final Map<String, LMPostViewData> _posts = {};
+
+  @override
+  void initState() {
+    super.initState();
+    userPostingRights = checkPostCreationRights();
+    addPageRequestListener();
+    _userCreatedCommentBloc.add(
```

Comment:
can remove _page key and this event as well, adding a page request listener will do the job